common/src/main/java/org/apache/cassandra/sidecar/common/ApiEndpointsV1.java

```diff
@@ -55,6 +57,8 @@ public final class ApiEndpointsV1
     public static final String SNAPSHOTS_ROUTE = API_V1 + PER_KEYSPACE + PER_TABLE + PER_SNAPSHOT;
     // Replaces DEPRECATED_COMPONENTS_ROUTE
     public static final String COMPONENTS_ROUTE = SNAPSHOTS_ROUTE + PER_COMPONENT;
+    public static final String COMPONENTS_WITH_INDEX_ROUTE_SUPPORT = SNAPSHOTS_ROUTE + PER_INDEX
```

Comment:
To be more clear, the name should indicate that it is for secondary index. The name `index` has been used else where in the other classes too. Please update them. 

Suggestions: `COMPONENTS_WITH_SI_ROUTE_SUPPORT` or `COMPONENTS_WITH_SECONDARY_INDEX_ROUTE_SUPPORT`

I think using `SI` is fine, as it is a common term in Cassandra world.